library/plugins/pushgateway/pushgateway-plugin/src/main/kotlin/com/cdsap/talaiot/plugin/pushgateway/PushgatewayConfigurationProvider.kt

```diff
@@ -17,20 +17,20 @@ class PushgatewayConfigurationProvider(
         val talaiotExtension = project.extensions.getByName("talaiot") as PushgatewayExtension
 
         talaiotExtension.publishers?.apply {
-            val logger = LogTrackerImpl(talaiotExtension.logger)
-            PushGatewayFormatter()
```

Comment:
Not sure if this declaration was needed, since it is not used I've removed it. Let me know if need to be reverted.

Reply:
it's ok